model/task/task.go

```diff
@@ -775,6 +775,9 @@ func (t *Task) UpdateDisplayTask() error {
 
 	t.Status = status
 	t.TimeTaken = timeTaken
+	if IsFinished(*t) {
```

Comment:
do I want to know why IsFinished isn't a method on *t?

Reply:
It is now!